### PR TITLE
[13.0][MIG] mgmtsystem_info_security_manual

### PR DESCRIPTION
**Pull request by flachica**
Migration to 13.0

### CODE REVIEW

**Review by dreispt — CHANGES_REQUESTED** [review]
If you rename the module, we should provide the migration to allow people an easy upgrade.
Let me know if you prefer to keep the original name or need help on the mig script.

**Comment by flachica** [comment]
> If you rename the module, we should provide the migration to allow people an easy upgrade.
> Let me know if you prefer to keep the original name or need help on the mig script.

I need help on the mig script. I think that the correct name is mgmtsystem_information_security. I prefer rename

**Comment by FernandoRomera** [comment]
View a example in https://apps.odoo.com/apps/modules/12.0/web_pwa_oca/

In case you previously installed web_pwa, run the following steps with odoo shell, after having installed openupgradelib:

> from openupgradelib import openupgrade
> openupgrade.update_module_names(env.cr, [('web_pwa', 'web_pwa_oca')], merge_modules=False)
> env.cr.commit()

**Comment by flachica** [comment]
> n [https://apps.odoo.com/apps/modules/12.0/web_pwa_oca/](url):

The link is broken?

**Comment by FernandoRomera** [comment]
Or this
https://github.com/OCA/l10n-brazil/blob/12.0/l10n_br_base/migrations/12.0.1.0.1/pre-migration.py

**Comment by flachica** [comment]
@FernandoRomera I include the script. Thanks

File: mgmtsystem_information_security/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "version": "13.0.1.0.0",
+    "author": "Savoir-faire Linux, Odoo Community Association (OCA)",
+    "website": "http://www.savoirfairelinux.com",
+    "license": "AGPL-3",
+    "category": "Generic Modules/Others",
+    "depends": ["mgmtsystem_manual"],
+    "data": ["data/information_security_manual.xml"],
+    "demo": [],
+    "installable": True,
+}
```
Comment (FernandoRomera) [inline mgmtsystem_information_security/__manifest__.py]: Missing:
 "external_dependencies": {"python": ["openupgradelib"]},

And in requirements.txt:
openupgradelib

**Review by FernandoRomera — CHANGES_REQUESTED** [review]

File: mgmtsystem_information_security/__manifest__.py (hunk 1 above)
Comment (flachica) [reply mgmtsystem_information_security/__manifest__.py]: Thanks for revision. Included yer

File: mgmtsystem_information_security/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Information Security Management System Manual",
+    "version": "13.0.1.0.0",
+    "author": "Savoir-faire Linux, Odoo Community Association (OCA)",
+    "website": "http://www.savoirfairelinux.com",
```
Comment (ivantodorovich) [inline mgmtsystem_information_security/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/management-system",
```

File: mgmtsystem_information_security/migrations/13.0.1.0.0/pre-migration.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    openupgrade.update_module_names(
+        env.cr,
+        [("mgmtsystem_info_security_manual", "mgmtsystem_information_security")],
+        merge_modules=True,
+    )
```
Comment (ivantodorovich) [inline mgmtsystem_information_security/migrations/13.0.1.0.0/pre-migration.py]: This won't work, I think..

On current deployments `mgmtsystem_information_security` won't be installed, so this script will never run.

I'm not sure what the process is to handle this, most probably a script in https://github.com/OCA/openugprade ?

in any case, is it really worth the trouble to rename the module?

Comment (flachica) [reply mgmtsystem_information_security/migrations/13.0.1.0.0/pre-migration.py]: All right. I will rename ASAP. More simple is more good. Thanks @ivantodorovich

Comment (flachica) [reply mgmtsystem_information_security/migrations/13.0.1.0.0/pre-migration.py]: Renamed! Thanks

**Review by FernandoRomera — APPROVED** [review]

**Review by AntoniRomera — APPROVED** [review]

**Comment by FernandoRomera** [comment]
@dreispt can you remove the request or review?

**Pull request merged**